S7.Net/Types/S7StringAttribute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace S7.Net.Types
+{
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
+    public sealed class S7StringAttribute : Attribute
+    {
+        private readonly S7StringType type;
+        private readonly int reservedLength;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="S7StringAttribute"/> class.
+        /// </summary>
+        /// <param name="type">The string type.</param>
+        /// <param name="reservedLength">Reserved length of the string in characters (default value is 254).</param>
+        /// <exception cref="ArgumentException">Please use a valid value for the string type</exception>
+        public S7StringAttribute(S7StringType type, int reservedLength = 254)
```

Comment:
Not a fan of the default length. Is this also the default when you add a string in TIA Portal? Personally I would prefer that you'd have to be explicit about this, so that you can't forget either.

Reply:
I made it like this because it is normally the default length of a string (also wstring) type in TIA. But you are right, that it is probably better to make it explicit.